### PR TITLE
Lighthouse: alts are missing on the home page

### PR DESCRIPTION
**Pull request by qmonmert**
Home page: https://www.jhipster.tech/

I added some `alt=""` because the `span` below is explicit and we avoid this error: `Image elements have [alt] attributes that are redundant text`

image

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -144,7 +144,7 @@
                     <div class="side-menu-container">
                         <div class="navbar-header">
                             <a class="navbar-brand" href="/">
-                                <div class="icon"><img src="{{ site.url }}/images/logo/logo-jhipster.svg" height="19px"></div>
+                                <div class="icon"><img src="{{ site.url }}/images/logo/logo-jhipster.svg" alt="Logo JHipster" height="19px"></div>
```
Comment (mraible) [inline _layouts/default.html:147]: I'd change this to "JHipster Logo"

File: index.html [hunk 2]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
         <div class="row stack-container no-margin">
             <ul class="text-center stacks">
                 <li class="clip img-circle">
-                    <img src="images/logo/svg/html-5.svg" />
+                    <img src="images/logo/svg/html-5.svg" alt="" />
```
Comment (mraible) [inline index.html:365]: Should this say HTML 5 Logo?

File: _layouts/default.html (hunk 1 above)
Comment (mraible) [inline _layouts/default.html:147]:
```suggestion
                                <div class="icon"><img src="{{ site.url }}/images/logo/logo-jhipster.svg" alt="JHipster" height="19px"></div>
```

**Comment by qmonmert** [comment]
thanks @mraible

File: index.html (hunk 2 above)
Comment (mraible) [inline index.html:365]:
```suggestion
                    <img src="images/logo/svg/html-5.svg" alt="HTML5 Logo" />
```

File: index.html [hunk 3]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">HTML5</span>
                 </li>
                 <li class="clip img-circle">
-                    <img src="images/logo/svg/css-3.svg" />
+                    <img src="images/logo/svg/css-3.svg" alt="" />
```
Comment (mraible) [inline index.html:369]:
```suggestion
                    <img src="images/logo/svg/css-3.svg" alt="CSS3 Logo" />
```

File: index.html [hunk 4]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">CSS3</span>
                 </li>
                 <li class="clip img-circle">
-                    <img src="images/logo/svg/bootstrap.svg" />
+                    <img src="images/logo/svg/bootstrap.svg" alt="" />
```
Comment (mraible) [inline index.html:373]:
```suggestion
                    <img src="images/logo/svg/bootstrap.svg" alt="Bootstrap" />
```

File: index.html [hunk 5]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Bootstrap</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/typescript.svg" />
+                    <img src="images/logo/svg/typescript.svg" alt="" />
```
Comment (mraible) [inline index.html:377]:
```suggestion
                    <img src="images/logo/svg/typescript.svg" alt="TypeScript" />
```

File: index.html [hunk 6]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">TypeScript</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/angular.svg" />
+                    <img src="images/logo/svg/angular.svg" alt="" />
```
Comment (mraible) [inline index.html:381]:
```suggestion
                    <img src="images/logo/svg/angular.svg" alt="Angular" />
```

File: index.html [hunk 7]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Angular</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/react.svg" />
+                    <img src="images/logo/svg/react.svg" alt="" />
```
Comment (mraible) [inline index.html:385]:
```suggestion
                    <img src="images/logo/svg/react.svg" alt="React" />
```

File: index.html [hunk 8]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">React</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/vue.svg" />
+                    <img src="images/logo/svg/vue.svg" alt="" />
```
Comment (mraible) [inline index.html:389]:
```suggestion
                    <img src="images/logo/svg/vue.svg" alt="Vue" />
```

File: index.html [hunk 9]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Vue</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/redux.svg" />
+                    <img src="images/logo/svg/redux.svg" alt="" />
```
Comment (mraible) [inline index.html:393]:
```suggestion
                    <img src="images/logo/svg/redux.svg" alt="Redux" />
```

File: index.html [hunk 10]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Redux</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/icons/websocket.png" class="invert"/>
+                    <img src="images/logo/icons/websocket.png" alt="" class="invert"/>
```
Comment (mraible) [inline index.html:397]:
```suggestion
                    <img src="images/logo/icons/websocket.png" alt="WebSocket" class="invert"/>
```

File: index.html [hunk 11]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Websockets</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/npm.svg"/>
+                    <img src="images/logo/svg/npm.svg" alt=""/>
                     <span class="title_sub_tech">Npm</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/webpack.svg"/>
+                    <img src="images/logo/svg/webpack.svg" alt=""/>
```
Comment (mraible) [inline index.html:405]:
```suggestion
                    <img src="images/logo/svg/webpack.svg" alt="webpack"/>
```

File: index.html [hunk 12]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Webpack</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/sass.svg" class="height" />
+                    <img src="images/logo/svg/sass.svg" alt="" class="height" />
```
Comment (mraible) [inline index.html:409]:
```suggestion
                    <img src="images/logo/svg/sass.svg" alt="Sass" class="height" />
```

File: index.html [hunk 13]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Sass</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/browsersync.svg" />
+                    <img src="images/logo/svg/browsersync.svg" alt="" />
```
Comment (mraible) [inline index.html:413]:
```suggestion
                    <img src="images/logo/svg/browsersync.svg" alt="Browsersync" />
```

File: index.html [hunk 14]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Browsersync</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/jest.svg" />
+                    <img src="images/logo/svg/jest.svg" alt="" />
```
Comment (mraible) [inline index.html:417]:
```suggestion
                    <img src="images/logo/svg/jest.svg" alt="Jest" />
```

File: index.html [hunk 15]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Jest</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/cypress.svg" />
+                    <img src="images/logo/svg/cypress.svg" alt="" />
```
Comment (mraible) [inline index.html:421]:
```suggestion
                    <img src="images/logo/svg/cypress.svg" alt="Cypress" />
```

File: index.html [hunk 16]
```diff
@@ -362,67 +362,67 @@ <h2 class="heading">Client side options</h2>
...
                     <span class="title_sub_tech">Cypress</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/icons/protractor.png" />
+                    <img src="images/logo/icons/protractor.png" alt="" />
                     <span class="title_sub_tech">Protractor</span>
                 </li>
```
Comment (mraible) [inline index.html:427]: Protractor should be removed.

File: index.html [hunk 17]
```diff
@@ -437,147 +437,147 @@ <h2 class="heading">Server side options</h2>
         <div class="row stack-container no-margin">
             <ul class="text-center stacks">
                 <li class="clip img-circle">
-                    <img src="images/logo/svg/spring-boot.svg" />
+                    <img src="images/logo/svg/spring-boot.svg" alt="" />
```
Comment (mraible) [inline index.html:440]:
```suggestion
                    <img src="images/logo/svg/spring-boot.svg" alt="Spring Boot" />
```

File: index.html [hunk 18]
```diff
@@ -437,147 +437,147 @@ <h2 class="heading">Server side options</h2>
...
                     <span class="title_sub_tech">Spring Boot</span>
                 </li>
                 <li class="clip img-circle">
-                    <img src="images/logo/icons/spring.png"/>
+                    <img src="images/logo/icons/spring.png" alt=""/>
```
Comment (mraible) [inline index.html:444]:
```suggestion
                    <img src="images/logo/icons/spring.png" alt="Spring Security"/>
```

File: index.html [hunk 19]
```diff
@@ -437,147 +437,147 @@ <h2 class="heading">Server side options</h2>
...
                     <span class="title_sub_tech">Spring Security</span>
                 </li>
                 <li class="clip img-circle">
-                    <img src="images/logo/svg/micronaut.svg" />
+                    <img src="images/logo/svg/micronaut.svg" alt="" />
```
Comment (mraible) [inline index.html:448]:
```suggestion
                    <img src="images/logo/svg/micronaut.svg" alt="Micronaut" />
```

File: index.html [hunk 20]
```diff
@@ -437,147 +437,147 @@ <h2 class="heading">Server side options</h2>
...
                     <span class="title_sub_tech">Micronaut</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/icons/netflix.png"/>
+                    <img src="images/logo/icons/netflix.png" alt=""/>
```
Comment (mraible) [inline index.html:452]:
```suggestion
                    <img src="images/logo/icons/netflix.png" alt="Netflix OSS"/>
```

File: index.html [hunk 21]
```diff
@@ -437,147 +437,147 @@ <h2 class="heading">Server side options</h2>
...
                     <span class="title_sub_tech">Netflix OSS</span>
                 </li>
                 <li class="clip img-circle new">
-                    <img src="images/logo/svg/consul.svg"/>
+                    <img src="images/logo/svg/consul.svg" alt=""/>
```
Comment (mraible) [inline index.html:456]:
```suggestion
                    <img src="images/logo/svg/consul.svg" alt="Consul"/>
```

**Comment by mraible** [comment]
> have you read the description of this PR before review it?

It looks like "" is on purpose. I'll close my suggestions.

**Review by atomfrede — APPROVED** [review]

**Pull request merged**